Makefile

```diff
@@ -58,11 +58,15 @@ $(TOOLS_DIR)/misspell: $(TOOLS_MOD_DIR)/go.mod $(TOOLS_MOD_DIR)/go.sum $(TOOLS_M
 	cd $(TOOLS_MOD_DIR) && \
 	go build -o $(TOOLS_DIR)/misspell github.com/client9/misspell/cmd/misspell
 
+$(TOOLS_DIR)/gosec: $(TOOLS_MOD_DIR)/go.mod $(TOOLS_MOD_DIR)/go.sum $(TOOLS_MOD_DIR)/tools.go
+	cd $(TOOLS_MOD_DIR) && \
+	go build -o $(TOOLS_DIR)/gosec github.com/securego/gosec/v2/cmd/gosec
+
 .PHONY: lint
-lint: $(TOOLS_DIR)/golangci-lint $(TOOLS_DIR)/misspell
+lint: $(TOOLS_DIR)/golangci-lint $(TOOLS_DIR)/misspell $(TOOLS_DIR)/gosec
 	$(TOOLS_DIR)/golangci-lint run --timeout=5m
-	$(TOOLS_DIR)/misspell -w $(ALL_DOCS) && \
-	go mod tidy
+	$(TOOLS_DIR)/misspell -w $(ALL_DOCS)
+	$(TOOLS_DIR)/gosec -exclude-dir=test/e2e ./...
```

Comment:
Why do we need to do this instead of adding `gosec` as a linter in the golangci-lint config?

Reply:
did not realize gosec is part of golangci-lint